src/system/Accordion/Accordion.test.tsx

```diff
@@ -48,9 +47,11 @@ describe( '<Accordion />', () => {
 	} );
 
 	it( 'should open the content when clicking on its trigger', async () => {
+		const user = userEvent.setup();
+
 		const { container } = renderComponent();
 
-		fireEvent.click( screen.getByRole( 'button', { name: 'trigger two' } ) );
+		await user.click( screen.getByRole( 'button', { name: 'trigger two' } ) );
 
 		// Should find the open content
 		expect( screen.queryByText( 'content one' ) ).toBeNull();
```

Comment:
Not a cause of this PR, but I noticed the comments are mixed up in the second test. It should be: 

```diff
diff --git a/src/system/Accordion/Accordion.test.tsx b/src/system/Accordion/Accordion.test.tsx
index cc9d93a..cbd460c 100644
--- a/src/system/Accordion/Accordion.test.tsx
+++ b/src/system/Accordion/Accordion.test.tsx
@@ -52,11 +52,11 @@ describe( '<Accordion />', () => {
 
-		// Should find the open content
+		// Should not find the closed content
 		expect( screen.queryByText( 'content one' ) ).toBeNull();
 
-		// Should not find the closed content
-		expect( screen.queryByText( 'content two' ) ).toBeInTheDocument();
+		// Should find the open content
+		expect( screen.getByText( 'content two' ) ).toBeInTheDocument();
```

We could also change the `queryBy` to `getBy` for the second assertion since I believe `queryBy` is usually used when we expect it not to be found.

Reply:
This could be a follow-up, but if we're already editing the test... 😄 we could make the test more robust by making it more specific and checking against things like the expanded/open state which would affect the user's experience. Also asserting [toBeVisible](https://github.com/testing-library/jest-dom#tobevisible), which checks if it's in the document like `toBeInTheDocument`, but it also checks to ensure it's not hidden via attributes or CSS. 

```diff
diff --git a/src/system/Accordion/Accordion.test.tsx b/src/system/Accordion/Accordion.test.tsx
index d770ccc..26d5f34 100644
--- a/src/system/Accordion/Accordion.test.tsx
+++ b/src/system/Accordion/Accordion.test.tsx
@@ -51,13 +51,20 @@ describe( '<Accordion />', () => {
 
 		const { container } = renderComponent();
 
-		await user.click( screen.getByRole( 'button', { name: 'trigger two' } ) );
-
-		// Should find the open content
-		expect( screen.queryByText( 'content one' ) ).toBeNull();
+		await user.click( screen.getByRole( 'button', { name: 'trigger two', expanded: false } ) );
 
 		// Should not find the closed content
-		expect( screen.queryByText( 'content two' ) ).toBeInTheDocument();
+		expect(
+			screen.getByRole( 'button', { name: 'trigger one', expanded: false } )
+		).toHaveAttribute( 'data-state', 'closed' );
+		expect( screen.queryByText( 'content one' ) ).not.toBeInTheDocument();
+
+		// Should find the open content
+		expect( screen.getByRole( 'button', { name: 'trigger two', expanded: true } ) ).toHaveAttribute(
+			'data-state',
+			'open'
+		);
+		expect( screen.getByText( 'content two' ) ).toBeVisible();
 
 		// Check for accessibility issues
 		expect( await axe( container ) ).toHaveNoViolations();
```